### PR TITLE
Fix whitespace in markdown

### PR DESCRIPTION
**Pull request by kagarmoe**
Signed-off-by: kagarmoe <kgarmoe@chef.io>

The Markdown formatting bothers me every time I write in these docs and I need to work on the install docs. My Markdown linter agrees, and it throws so many errors that I can't see what is important. I'm planning on adding this linter (with some customizations) to the docs as a test later this year.

## Description
1. Use one space after terminal punctuation
1. Let Hugo handle text wrapping in definition lists

Did I regret this decision 20 files in? Yes, yes I did.

### CODE REVIEW

File: docs-chef-io/content/workstation/ctl_kitchen.md [hunk 1]
```diff
@@ -1079,18 +1016,18 @@ This subcommand has the following options:
 
 `-c`, `--concurrency`
 
-:   The number of allowed concurrent connections. Default: `9999` (all
+: The number of allowed concurrent connections. Default: `9999` (all
```
Comment (kagarmoe) [inline docs-chef-io/content/workstation/ctl_kitchen.md:1019]: Missed a few.

File: docs-chef-io/content/workstation/knife_cookbook.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ This argument has the following options:
 
 `-p`, `--purge`
 
-:   Entirely remove a cookbook (or cookbook version) from the Chef Infra
+: Entirely remove a cookbook (or cookbook version) from the Chef Infra
```
Comment (kagarmoe) [inline docs-chef-io/content/workstation/knife_cookbook.md:46]: Missed a few more.

File: docs-chef-io/content/workstation/knife_cookbook.md [hunk 3]
```diff
@@ -309,36 +309,36 @@ This argument has the following options:
 
 `COOKBOOK_VERSION`
 
-:   The version of a cookbook to be shown. If a cookbook has only one
+: The version of a cookbook to be shown. If a cookbook has only one
```
Comment (kagarmoe) [inline docs-chef-io/content/workstation/knife_cookbook.md:312]: Missed these

File: docs-chef-io/content/workstation/knife_environment.md [hunk 4]
```diff
@@ -150,7 +150,7 @@ This argument has the following options:
 
 `--description DESCRIPTION`
 
-:   The description of the environment. This value populates the
+: The description of the environment. This value populates the
```
Comment (kagarmoe) [inline docs-chef-io/content/workstation/knife_environment.md:153]: Missed this one, too.

File: docs-chef-io/content/workstation/knife_serve.md [hunk 5]
```diff
@@ -35,22 +35,22 @@ This subcommand has the following options:
...
 `--chef-zero-host IP`
 
-:   Override the host on which chef-zero listens. Default value:
+: Override the host on which chef-zero listens. Default value:
     `127.0.0.1`.
 
 `--chef-zero-port PORT`
 
-:   The port on which chef-zero listens. The default behavior will bind
+: The port on which chef-zero listens. The default behavior will bind
```
Comment (kagarmoe) [inline docs-chef-io/content/workstation/knife_serve.md:48]: Missed these two

File: docs-chef-io/content/workstation/knife_supermarket.md [hunk 6]
```diff
@@ -262,20 +249,20 @@ where `mysql` is the search term. This will return something similar to:
 
 ``` bash
 mysql:
-  cookbook:             https://supermarket.chef.io/api/v1/cookbooks/mysql
+  cookbook:           https://supermarket.chef.io/api/v1/cookbooks/mysql
```
Comment (kagarmoe) [inline docs-chef-io/content/workstation/knife_supermarket.md:252]: Undo these changes.

File: docs-chef-io/content/workstation/plugin_knife_opc.md [hunk 7]
```diff
@@ -238,16 +238,16 @@ wq
 Saved arno.
 knife opc user show arno
 display_name: arno schmidt
-email:        arno@opscode.io
-first_name:   arno
-last_name:    schmidt
+email:      arno@opscode.io
```
Comment (kagarmoe) [inline docs-chef-io/content/workstation/plugin_knife_opc.md:241]: undo these

File: docs-chef-io/content/workstation/plugin_knife_opc.md [hunk 8]
```diff
@@ -325,8 +325,8 @@ knife opc org show ORG_NAME
 ``` bash
 knife opc org show acme
 full_name: Acme
-guid:      cc9f9d0d4f6e7e35272e327e22e7affc
-name:      acme
+guid:    cc9f9d0d4f6e7e35272e327e22e7affc
```
Comment (kagarmoe) [inline docs-chef-io/content/workstation/plugin_knife_opc.md:328]: undo

File: docs-chef-io/content/workstation/plugin_knife_opc.md [hunk 9]
```diff
@@ -383,8 +383,8 @@ knife opc org delete ORG_NAME
 knife opc org delete acme2
 Do you want to delete the organization acme2? (Y/N) Y
 full_name: The Other Acme
-guid:      2adec1140cf777a15d82d9099304da71
-name:      acme2
+guid:    2adec1140cf777a15d82d9099304da71
```
Comment (kagarmoe) [inline docs-chef-io/content/workstation/plugin_knife_opc.md:386]: undo

File: docs-chef-io/content/workstation/config_rb.md [hunk 10]
```diff
@@ -85,76 +85,76 @@ This configuration file has the following settings:
...
-    data_bag_encrypt_version 2
-    ```
+  ``` ruby
+  data_bag_encrypt_version 2
+  ```
 
 `fips`
 : Allows OpenSSL to enforce FIPS-validated security during a Chef Infra Client run. Set to `true` to enable FIPS-validated security.
 
 The following operating systems are supported:
```
Comment (IanMadd) [inline docs-chef-io/content/workstation/config_rb.md:105]: I think this and the following list should be indented to 3 spaces.

```suggestion
   The following operating systems are supported:
```

File: docs-chef-io/content/workstation/config_rb.md [hunk 11]
```diff
@@ -85,76 +85,76 @@ This configuration file has the following settings:
...
 
-    - Red Hat Enterprise Linux
-    - Oracle Enterprise Linux
-    - CentOS
-    - Windows
+- Red Hat Enterprise Linux
+- Oracle Enterprise Linux
+- CentOS
+- Windows
```
Comment (IanMadd) [inline docs-chef-io/content/workstation/config_rb.md:110]:
```suggestion
   - Red Hat Enterprise Linux
   - Oracle Enterprise Linux
   - CentOS
   - Windows
```

File: docs-chef-io/content/workstation/config_rb.md (hunk 10 above)
Comment (kagarmoe) [reply docs-chef-io/content/workstation/config_rb.md:105]: Thanks!

File: docs-chef-io/content/workstation/config_rb.md (hunk 11 above)
Comment (kagarmoe) [reply docs-chef-io/content/workstation/config_rb.md:110]: Two spaces appears to work.

**Review by IanMadd — APPROVED** [review]
tenor-37061229
